SwanGallery/setup.py

```diff
@@ -0,0 +1,111 @@
+"""
+SwanGallery setup
+"""
+import json
+import sys
+from pathlib import Path
+
+import setuptools
+
+HERE = Path(__file__).parent.resolve()
+
+# Get the package info from package.json
+pkg_json = json.loads((HERE / "package.json").read_bytes())
+
+# The name of the project
+name = "SwanGallery"
+
+lab_path = (HERE / pkg_json["jupyterlab"]["outputDir"])
+
+# Representative files that should exist after a successful build
+ensured_targets = [
+    str(lab_path / "package.json"),
+    str(lab_path / "static/style.js")
+]
+
+labext_name = pkg_json["name"]
+
+data_files_spec = [
+    ("share/jupyter/labextensions/%s" % labext_name, str(lab_path.relative_to(HERE)), "**"),
+    ("share/jupyter/labextensions/%s" % labext_name, str("."), "install.json"),
+    ("etc/jupyter/jupyter_server_config.d",
+     "jupyter-config/server-config", "SwanGallery.json"),
+    # For backward compatibility with notebook server
+    ("etc/jupyter/jupyter_notebook_config.d",
+     "jupyter-config/nb-config", "SwanGallery.json"),
+]
+
+long_description = (HERE / "README.md").read_text(encoding="utf8")
+
+version = (
+    pkg_json["version"]
```

Comment:
Can you configure bump2version similar to [other extensions in this repo](https://github.com/swan-cern/jupyter-extensions/blob/master/HdfsBrowser/.bumpversion.cfg).

The python package is built and pushed to pypi with a new version automatically using a [release pipeline](https://github.com/swan-cern/jupyter-extensions/blob/c309c0caf7494befbf9ed8ef9b6beed1e09a3f00/.github/workflows/release.yml#L34) documented in the [README](https://github.com/swan-cern/jupyter-extensions#create-a-release). So we don't have to manually upgrade versions.

https://pypi.org/project/bump2version/